scripts/upgrade

```diff
@@ -80,62 +85,65 @@ fi
 
 # Download, check integrity, uncompress and patch the source from app.src
 ynh_setup_source "$final_path"
-sudo cp -a "$tmpdir/store" "${final_path}"
-sudo cp -a "$tmpdir/.htconfig.php" "${final_path}"
-sudo cp -a "$tmpdir/php.log" "${final_path}"
-sudo rm -Rf "$tmpdir"
-sudo chmod -R 777 $final_path/store
-sudo mkdir $final_path/addon
-ynh_setup_source "$final_path/addon" "app_addons"
+cp -a "$tmpdir/store" "${final_path}"
+cp -a "$tmpdir/.htconfig.php" "${final_path}"
+cp -a "$tmpdir/php.log" "${final_path}"
+cp -a "$tmpdir/cache" "${final_path}"
+rm -Rf "$tmpdir"
+chmod -R 777 $final_path/store
+chmod -R 777 $final_path/cache
```

Comment:
My god ... please no chmod 777 ... I know this is from the previous code but what the hell is this needed for ...

Reply:
What would you suggest? It did not worked last time.